addons/im_livechat/static/src/embed/common/close_confirmation.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="im_livechat.closeConfirmation">
+        <div
+            class="o-livechat-closeConfirmation-overlay z-1 bg-black-50 position-absolute w-100 h-100 d-flex justify-content-center align-items-center"
+            t-on-keydown.capture.stop="onKeydown"
+            t-on-click.stop="onClick"
+            t-ref="closeConfirmation"
+            tabindex="1"
+        >
+            <div class="o-livechat-closeConfirmation-dialog rounded bg-white bg-opacity-100 p-3 m-3 d-flex flex-column">
+                <div class="d-flex justify-content-end">
+                    <button class="o-livechat-closeConfirmation-close btn-close" />
```

Comment:
Please follow BEM convention in Discuss code:

- Template is named `moduleName.ComponentName`, here `im_livechat.CloseConfirmation`
- Root node is named `o-moduleName-ComponentName`, here `o-livechat-CloseConfirmation`
- Parts are named `o-moduleName-ComponentName-partName`, here `o-livechat-CloseConfirmation-dialog`